### PR TITLE
Add release workflow

### PR DESCRIPTION
**Pull request by tiagovtristao**

### CODE REVIEW

File: .appveyor/appveyor.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '{branch}.{build}'
 branches:
   only:
   - v1
-clone_folder: c:\gopath\src\gopkg.in\gcfg.v1
+clone_folder: c:\gopath\src\gopkg.in\please-build\gcfg.v1
```
Comment (Tatskaari) [inline .appveyor/appveyor.yml]: What's thing thing? we can probably get rid of it.

File: .github/workflows/release.v1.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: tatskaari/release-action@master
+        with:
+          release-files: .
```
Comment (Tatskaari) [inline .github/workflows/release.v1.yaml:17]: Can you just get rid of this? If not, I should fix that.

**Review by Tatskaari — APPROVED** [review]
Looks good to me.

File: .appveyor/appveyor.yml (hunk 1 above)
Comment (tiagovtristao) [reply .appveyor/appveyor.yml]: No idea. I'll remove it if we are not using it

File: .github/workflows/release.v1.yaml (hunk 2 above)
Comment (tiagovtristao) [reply .github/workflows/release.v1.yaml:17]: I can't tell. Any way of testing this before merge?

**Pull request merged**